rts/Lua/LuaUnsyncedCtrl.cpp

```diff
@@ -2159,6 +2160,35 @@ int LuaUnsyncedCtrl::SetUnitNoMinimap(lua_State* L)
 	return 0;
 }
 
+/***
+ * @function Spring.SetMiniMapRotation
+ * @param rotation radians
```

Comment:
Is `radians` a type? Perhaps this should be something like "number radians". @rhys-vdw